src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+
+Les recommandations pour réduire les coûts de performance du chargement des polices web à partir d'un autre hôte comprennent l'utilisation des fonctions `preconnect`, `dns-prefetch` et `preload` [resource hints](./resource-hints), mais les polices web hautement prioritaires devraient être fournies par le même hôte afin de minimiser leur impact sur les performances. Ceci est particulièrement important pour les polices utilisées par des contenus à prééminence visuelle ou pour des corps de texte occupant la majorité d'une page.
+
+{{ figure_markup(
+  image="fig1.png",
+  caption="Les stratégies d'hébergement de polices web les plus populaires.",
+  description="Graphique à barres montrant la popularité des stratégies d'hébergement tierces-parties et d'auto-hébergement pour les polices web. 75% des pages web mobiles utilisent des hébergements tierces-parties et 25% des auto-hébergements. Les sites web en version ordinateur de bureau ont une utilisation similaire.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQDogXDb3BwZZHrBT39qccP_LJoCScD3QEi_FmjT_8VDPD_1Srpz-g7ZuuTUEb8pYXBpDmQzZ1hQh7q/pubchart?oid=1546332659&format=interactive"
+  )
+}}
+
+Le fait que les trois quarts soient hébergés sur des serveurs tierces-parties n'est peut-être pas surprenant compte tenu de la domination de Google Fonts dont nous parlerons [ci-dessous](#quels-sont-les-hébergeurs-tierces-parties-les-plus-populaires-).
+
+Google fournit des polices en utilisant des fichiers CSS tiers hébergés sur https://fonts.googleapis.com. Les développeurs font des requêtes vers ces feuilles de style en utilisant les balises `<link>` dans leur code. Bien que ces feuilles de style soient bloquantes, elles sont très petites. Cependant, les fichiers de police sont hébergés sur un autre domaine, `https://fonts.gstatic.com`. Le modèle consistant à exiger deux rebonds séparés vers deux domaines différents fait de `preconnect` une excellente option ici pour la deuxième requête qui ne sera pas découverte avant le téléchargement de la feuille de style.
+
+Notez que si `preload` pourrait être un ajout intéressant pour charger les fichiers de police plus haut dans la cascade de requêtes (rappelez-vous que `preconnect` établit la connexion, il ne demande pas le contenu du fichier), `preload` n'est pas encore disponible avec Google Fonts. Google Fonts génère des URL uniques pour leurs fichiers de police <a hreflang="en" href="https://github.com/google/fonts/issues/1067">qui sont susceptibles de changer</a>.
+
+### Quels sont les hébergeurs tierces-parties les plus populaires ?
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Hébergeur</th>
```

Comment:
Un petit `scope="col"` sur les `th` please :)

Reply:
I tend to add those only for complex tables where the scope is not obvious. Not sure if it makes a difference for simple tables?

Reply:
It's a bit better for vocalization, better to be more explicit in all cases :)

Reply:
Fixed.